docs/articles/documentation/guides/advanced-guides/authentication.md

```diff
@@ -1,50 +1,37 @@
 ---
 layout: docs
-title: User Roles
-permalink: /documentation/test-api/authentication/user-roles.html
+title: Authentication
+permalink: /documentation/guides/advanced-guides/authentication.html
 ---
-# User Roles
+# Authentication
+
+TestCafe ships with the [user roles](#user-roles) mechanism that covers most authentication types that require user actions on the page.
```

Comment:
Do you mean:
TestCafe allows you to specify [user roles](#user-roles) to test user interaction based on different authentication types.

If not, please clarify.

Reply:
First, I'll explain `authentication types that require user actions on the page`.

That's basically any authentication users come across on the web. Like, when they type a login, password and click "Sign In". Or when they click "Sign in with Facebook". These are all authentication types that require some user interaction. To perform these kinds of authentication in tests, you should use roles.

However, TestCafe also supports HTTP Basic and NTLM authentication (mentioned in the next paragraph) that happen behind the scenes. That's when roles won't help.

Reply:
But I maybe we shouldn't try to explain this in the introduction, but instead, rewrite this and the next paragraph like this:

*TestCafe ships with the [user roles](#user-roles) mechanism that allows you to emulate user actions to log in on a website. You can also use [HTTP Basic and NTLM authentication](#http-authentication).*

Reply:
Thank you. Let's change it to:
TestCafe includes a user role mechanism that allows you to emulate user actions to log in to a website. You can also use HTTP Basic and NTLM authentication.